tests/foreman/cli/test_errata.py

```diff
@@ -238,11 +416,21 @@ def test_negative_install_without_hc_info(self):
 
         @Assert: Error message thrown.
 
-        @caseautomation: notautomated
-
         """
+        with self.assertRaises(CLIReturnCodeError):
+            HostCollection.erratum_install({
+                'organization-id': self.org['id'],
+                'errata': [self.CUSTOM_ERRATA_ID]
+            })
+        # because of the bug we cannot verify the output of the error message
+        # it seems that hammer verify errata resource before host-collection
+        # verify that the errata package is not installed
+        for virtual_machine in self.virtual_machines:
+            self.assertFalse(
+                self._is_errata_package_installed(virtual_machine))
```

Comment:
Imo this check is redundant. You already ensured exception was risen, that should be good enough. Verifying package wasn't installed in this case doesn't make much sense to me as you haven't passed any host/hostcollection info, so there's no way sat could know where to install the package. @oshtaier your thoughts ?